sdk/keyvault/azure-security-keyvault/tests/test_secrets_client.py

```diff
@@ -77,52 +77,36 @@ def test_secret_crud_operations(self, vault_client, **kwargs):
         self.assertEqual(expires, created.expires)
         self.assertEqual(tags, created.tags)
 
-        # get secret without version
-        self._assert_secret_attributes_equal(created, client.get_secret(created.name, ""))
-
-        # get secret with version
+        self._assert_secret_attributes_equal(created, client.get_secret(created.name))
         self._assert_secret_attributes_equal(created, client.get_secret(created.name, created.version))
 
         def _update_secret(secret):
```

Comment:
can we remove `_update_secret` method out of the `test_secret_crud_operations`.

Reply:
I'll leave that to a future cleanup, my goal here is consistent live test runs.